abpframework/abp
# Update `datatables.net` to `2.0.2`.

**The milestone is `8.3`, we can review and test it later.**


Resolve #19323

How to test:

1. Add `resolutions` section to `packages.json` and restore packages.

```
"resolutions": {
  "datatables.net": "^2.0.2",
  "datatables.net-bs5": "^2.0.2"
}
```

2. Update the `abp.resourcemapping.js` on the template web project.
```
module.exports = {
    aliases: {

    },
    clean: [

    ],
    mappings: {
        "@node_modules/datatables.net/js/dataTables.min.js": "@libs/datatables.net/js/"
    }
};
```

3.  Run web project and check if all `datatables` work.
